tests/test_tokenization_clip.py

```diff
@@ -206,3 +164,65 @@ def test_add_tokens_tokenizer(self):
                 # padding is very hacky in CLIPTokenizer, pad_token_id is always 0
                 # so skip this check
                 # self.assertEqual(tokens[-2], tokenizer.pad_token_id)
+
+    def test_check_encoding_slow_fast(self):
```

Comment:
Comment 3

Reply:
I added a test to be able to test cases which according to me should not be tokenized differently between the slow version and the fast version of the tokenizer